src/pages/vaults/VaultPositionsTable.tsx

```diff
@@ -55,6 +56,8 @@ export const VaultPositionsTable = ({ className }: { className?: string }) => {
           label: stringGetter({ key: STRING_KEYS.MARKET }),
           renderCell: ({ marketId, currentLeverageMultiple }) => {
             const asset = marketId != null ? marketIdToAssetMetadataMap[marketId] : undefined;
+            const logoUrl =
```

Comment:
can we make sure USDC logo gets rendered somehow? The old asset icon knew how to render it (via the hardcoded one I guess)

Reply:
Of course!

Reply:
Added a comment as well